runtime/compiler/x/codegen/J9CodeGenerator.cpp

```diff
@@ -353,6 +353,29 @@ J9::X86::CodeGenerator::endInstructionSelection()
       }
    }
 
+TR::VectorLength
+J9::X86::CodeGenerator::getMaxPreferredVectorLength()
+   {
+   TR::CPU *cpu = &self()->comp()->target().cpu;
+
+   // TODO: More research is needed
```

Comment:
I don't like TODO's without an accompanying issue.  Please either create an issue to track the work or remove the TODO from the code.